pkgs/applications/graphics/paraview/default.nix

```diff
@@ -18,11 +18,15 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
-   cmakeFlags = [
-     "-DPARAVIEW_ENABLE_PYTHON=ON"
-     "-DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON"
-     "-DPARAVIEW_ENABLE_EMBEDDED_DOCUMENTATION=OFF"
-   ];
+  # numpy is needed for the Python filters
+  pythonWithNumpy = python.withPackages (ps: with ps; [ numpy ]);
```

Comment:
The name `pythonEnv` is used more commonly.

Note though, that because in the wrapper we need to set `pythonsitePackages`, there is actually no need anymore to create such an env.

Reply:
Ok, I probably misunderstand something then: I thought we need pythonEnv both to add numpy as a dependency so it's available anywhere at all, and then needed to set the PYTHONPATH so that paraview & friends look into the correct `side-packages` directory to actually find it.

Reply:
During build time numpy can be found by simply adding it to buildInputs`. For runtime, simply adding the specific site-packages to PYTHONPATH is sufficient as paraview already knows about CPython site-packages.

Reply:
Ok, I think I understand and I'll try to fix up my commit a bit more.